cmd/download.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var downloadCmd = &cobra.Command{
+	Use:   "download",
+	Short: "Download a Document",
+	Long:  "Download a remote document from a paperless server",
+
+	Run: func(cmd *cobra.Command, args []string) {
+		log.Debugf("Called Download with args %v", args)
+		if len(args) < 1 {
+			log.Debugf("Missing filename to download")
```

Comment:
This is Debug() so you won't see the error message unless the `-d` flag was passed. Consider Info() instead.

Reply:
👍